specs/latest/1.0/index.html

```diff
@@ -730,6 +730,12 @@ <h3>Types</h3>
 // The power preference settings are documented in the WebGLContextAttributes
 // section of the specification.
 enum WebGLPowerPreference { "default", "low-power", "high-performance" };
+
+enum WebGLPixelFormat {
```

Comment:
I think we should start off by defining this enum in the WebGL 2.0 spec, where SRGB8_ALPHA8 and RGBA16F textures are included in the core spec, even though renderability of RGBA16F is still an extension there. The interactions are more minimal, and this could be promoted to the WebGL 1.0 spec once it's been prototyped and better understood. Would you consider that?

Reply:
I don't think it's actually easier to go webgl1 first, particularly given rgba16f on webgl2.

Reply:
Let's discuss on #3222 , which allows feature detection and requires the user to explicitly enable the required extensions. If we agree there, then we can sidestep this disagreement.